core-bundle/src/Resources/contao/modules/Module.php

```diff
@@ -257,7 +257,12 @@ abstract protected function compile();
 	 */
 	protected function getResponseCacheTags(): array
 	{
-		return array(System::getContainer()->get(EntityCacheTags::class)->getTagForModelClass($this->objModel));
+		if (!$this->objModel instanceof Model)
```

Comment:
Why don't we check for `null` here? 
We should probably also adjust the typehint in L104 and return type of `getModel` to `?ModuleModel`, no?

Reply:
Because the `getTagForModelInstance()` method expects a `Model` object.

Reply:
Yes sure, but how can `$this->objModel` be anything else than a `Model` or `null` (if not defined). It gets set in the constructor where the type is checked.

Reply:
Changed in 60f279d3d835d53335127c8ac0e06be80ffc4d8a.